src/magento/magento-marketplace-synchronize-store.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Synchronize Your Store
+group: getting-started
+---
+
+{: .bs-callout-info}
+If your Magento installation is hosted on Adobe Commerce Cloud, see [Install, manage, and upgrade extensions][1].
+
+The Web Setup System Configuration tool uses the public and private keys from your Marketplace [profile]({% link magento/magento-marketplace-credentials.md %}) to synchronize your store with Magento Marketplace. After your store is synchronized, the services remain in sync so you can easily add extensions, as well as keep track of all extensions you have purchased and installed.
+
+## Step 1: Retrieve Your Marketplace Credentials
```

Comment:
Using sentence case/capitalization for H2 and up to adhere to devdocs and Adobe guidelines
```suggestion
## Step 1: Retrieve your Marketplace credentials
```